sk0-front/src/app/common/services/notification.service.ts

```diff
@@ -12,10 +12,9 @@ export class NotificationService {
 
   async toastMessage(message: string, position?: "top" | "middle" | "bottom") {
     const toast = await this.toastController.create({
-      color: "primary",
+      position: position ?? "top",
       message,
-      duration: 1500,
-      position
+      duration: 1500
     });
     await toast.present();
   }
```

Comment:
Shouldn't we rather add a default in the parameters?
```ts
async toastMessage(message: string, position?: "top" | "middle" | "bottom" = "top") {
    const toast = await this.toastController.create({
      position,
      message,
      duration: 1500
    });
    await toast.present();
  }
```

Reply:
corrected